src/app/theme/cake-theme.ts

```diff
@@ -0,0 +1,27 @@
+import { Theme } from "./symbols";
+
+export const cakeTheme: Theme = {
+  key: "cake",
+  name: "Cake Theme",
+  properties: {
+    "--background": "#f8edeb",
+    "--text": "#47205c",
+    "--grey": "#6d4e75",
+    "--secondary": "#f3e1e1",
+    "--secalt": "#fae1dd",
+    "--textalt": "#a97aa2",
+    "--norm": "#47205c",
+    "--formbg": "#fff5f3",
+    "--link": "#009aff",
+    "--hover": "#008cad",
+    "--border": "#fae1dd",
+    "--mborder": "#fae1dd",
+    "--filter": "blur(0.6px) grayscale(0.5) sepia(1) hue-rotate(227deg) saturate(9.2) opacity(0.78)",
```

Comment:
```suggestion
    "--filter": "grayscale(0.5) sepia(1) hue-rotate(227deg) saturate(9.2) opacity(0.78)",
```
the blur seems unnecessary and just makes the logo look strange.